types/resource_dereferencing.go

```diff
@@ -1,10 +1,10 @@
 package types
 
 type ResourceDereferencing struct {
-	Context               string                     `json:"@context,omitempty" example:"https://w3id.org/did-resolution/v1"`
-	DereferencingMetadata DereferencingMetadata      `json:"dereferencingMetadata"`
-	ContentStream         ContentStreamI             `json:"contentStream"`
-	Metadata              ResolutionResourceMetadata `json:"contentMetadata"`
+	Context               string                `json:"@context,omitempty" example:"https://w3id.org/did-resolution/v1"`
```

Comment:
Note that this header should only be inserted if the Accept header has some combination of LD+JSON (i.e., they have requested JSON-LD).

Reply:
Yep. This is handled already